src/xo-app/backup/overview/index.js

```diff
@@ -30,12 +49,105 @@ const getJobValues = job => job.values || job.items
 
 // ===================================================================
 
+const LOGS_PER_PAGE = 10
+
+@connectStore(() => {
+  const object = createGetObject((_, props) => props.value)
+  return (state, props) => ({object: object(state, props)})
+})
+class JobValue extends Component {}
+
+class JobParam extends JobValue {
+  render () {
+    const {
+      object,
+      paramKey,
+      paramValue
+    } = this.props
+    const displayKey = object && object.type || paramKey
+    const displayValue = object && (object.name_label || object.name) || paramValue
+    return <span><strong>{displayKey}:</strong> {displayValue} </span>
+  }
+}
+
+class JobReturn extends JobValue {
+  render () {
+    const {
+      object,
+      returnValue
+    } = this.props
+    let xoName = object && (object.name_label || object.name) && (xoName += object.type && ` (${object.type})` || '')
+    return <span> <Icon icon='arrow-right' /> {xoName || String(returnValue)}</span>
+  }
+}
+
+class Log extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      seeCalls: false
+    }
+  }
+
+  _toggleCalls = () => this.setState({seeCalls: !this.state.seeCalls})
+  _delete = id => deleteJobsLog(id).then(() => subscribeJobsLogs.forceRefresh())
+  render () {
+    const { log } = this.props
+    const { seeCalls } = this.state
+    return <tbody>
+      <tr>
+        <td>
+          <button type='button' onClick={this._toggleCalls} className={classnames('btn', 'btn-sm', {'btn-default': !log.hasErrors, 'btn-danger': log.hasErrors})}><Icon icon={seeCalls ? 'caret-up' : 'caret'} /></button>
+          {' '}
+          {log.jobId}
+        </td>
+        <td>{jobKeyToLabel[log.key]}</td>
+        <td><FormattedDate value={new Date(+log.start)} month='long' day='numeric' year='numeric' hour='2-digit' minute='2-digit' second='2-digit' /></td>
+        <td><FormattedDate value={new Date(+log.end)} month='long' day='numeric' year='numeric' hour='2-digit' minute='2-digit' second='2-digit' /></td>
+        <td>{moment.duration(+log.duration).humanize()}</td>
```

Comment:
It would be nice to isolate in a dedicate component and to know how to translate.